test/sql/14-copy-utility.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE company
```

Comment:
This file is testing \copy directive while  test/output/copy-utility.source tests COPY SQL command (although I certainly understand that \copy is tranformed into COPY FROM STDIN).